src/base/system_parameter.c

```diff
@@ -6126,6 +6133,18 @@ static SYSPRM_PARAM prm_Def[] = {
    (char *) NULL,
    (DUP_PRM_FUNC) NULL,
    (DUP_PRM_FUNC) NULL},
+  {PRM_ID_LOG_RECOVERY_REDO_WORKER_COUNT,
+   PRM_NAME_LOG_RECOVERY_REDO_WORKER_COUNT,
+   (PRM_FOR_SERVER | PRM_HIDDEN),
```

Comment:
Should be visible to users.
```suggestion
   (PRM_FOR_SERVER),
```

Reply:
Done